pages/payment.js

```diff
@@ -0,0 +1,281 @@
+import Head from 'next/head';
+import Header from '../components/Header.js';
+import Footer from '../components/Footer.js';
+import Link from 'next/link';
+import cookie from 'js-cookie';
+import React from 'react';
+import nextCookies from 'next-cookies';
+import { useState, useEffect } from 'react';
```

Comment:
This `useState` import could be combined with the other import from `react` above. The `useEffect` import is not needed.